### DISCUSSION TITLE
[In progress] Support for Espruino

### DISCUSSION THREAD

**Discussion by mfucci** [mfucci/node-matter#67] [Workstream]
The goal is to be able to node-matter on a [Espruino device](https://www.espruino.com/) (Microcontroller running js).

Most of the work will be to port Crypto.ts.
And probably renaming node-matter to ts-matter to make it clear that it is not dependent on node.js anymore.

Contributor:

- @adminy

**Comment by adminy** [comment]
This is going to be fun :)

**Comment by mfucci** [comment]
@adminy : I notified the Espruino community of this project and some have concerns about the RAM / ROM footprint and the connectivity (mostly BLE supported) for the current Espruino boards.
https://forum.espruino.com/conversations/380585/#comment16722318

What board were you planning on supporting?
Maybe you can reply directly on that thread.

**Reply by adminy** [reply]
Yes, I'll reply by the end of the day. Nrf52840 is the board is the one I'm playing with. I've bluetooth working so far, I'll come back with more details a little later.
